examples/DotNetCore/WebDemoWithEventHub/WebDemoWithEventHub/Program.cs

```diff
@@ -39,12 +49,17 @@ public static void Main(string[] args)
                             builder.AddAzureAppConfiguration(options =>
                             {
                                 options.Connect(new Uri(appConfigurationEndpoint), new DefaultAzureCredential())
+                                        .ConfigureKeyVault(kv =>
+                                        {
+                                            kv.SetCredential(new DefaultAzureCredential());
+                                        })
                                        .Select(keyFilter: "WebDemo:*")
                                        .ConfigureRefresh((refreshOptions) =>
                                        {
-                                           // Indicates that all configuration should be refreshed when the given key has changed.
-                                           refreshOptions.Register(key: "WebDemo:Sentinel", refreshAll: true);
+                                           refreshOptions.Register(key: "WebDemo:Settings", refreshAll: true);
```

Comment:
Why is this change?

Reply:
Undid this.

---
In reply to: [427590805](https://github.com/Azure/AppConfiguration/pull/329#discussion_r427590805) [](ancestors = 427590805)